src/api/app/services/workflows/yaml_downloader.rb

```diff
@@ -15,46 +15,42 @@ def call
     private
 
     def download_yaml_file
```

Comment:
This is just `.call`...

Reply:
Yeah, removing this method and adding it to `call` makes sense. Done